pkg/debug/scripts/collect-info.sh

```diff
@@ -16,9 +16,10 @@ VERSION=26
 PKG_DEPS="procps tar dmidecode iptables dhcpcd"
 
 DATE=$(date "+%Y-%m-%d-%H-%M-%S")
-INFO_DIR="eve-info-v$VERSION-$DATE"
-TARBALL_FILE="/persist/$INFO_DIR.tar.gz"
+INFO_DIR_SUFFIX="eve-info/v$VERSION-$DATE"
```

Comment:
Erik, can we please keep the "eve-info" prefix to the file itself? When only one file is sent or attached to the issue it will be difficult to understand what actually it means having only version.

Reply:
Done.